### ISSUE TITLE
Evaluate difficulty to migrate website to Carpentries redisigned webpage

### ISSUE THREAD

**Issue by MathewBiddle** [ioos/bio_mobilization_workshop#139]
To make our materials more accessible, we should try to migrate to The Carpentries redesigned lesson webpages. See https://carpentries.github.io/workbench/

As an example, see https://datacarpentry.org/R-ecology-lesson/index.html

There is a transition guide too https://carpentries.github.io/workbench/transition-guide.html

**Comment by MathewBiddle** [comment]
https://carpentries.github.io/sandpaper-docs/migrating-from-styles.html

**Comment by MathewBiddle** [comment]
keep an eye on 
* https://github.com/carpentries-lab/python-aos-lesson/issues/61

**Comment by MathewBiddle** [comment]
testing at https://github.com/MathewBiddle/test_workshop. See the test page at https://mathewbiddle.github.io/test_workshop/01-introduction.html

@laurabrenskelle lmk what you think so far.

xref: https://github.com/ioos/mbon_data_workshop_2025/issues/1

**Comment by laurabrenskelle** [comment]
I think I like this layout, but is there a way to change it so the solutions and tips are hidden in a dropdown that you click? Like this: https://carpentries.github.io/sandpaper-docs/example.html

Maybe this is the plan and it is currently just missing something on the backend to implement this?

[This page](https://ioos.github.io/bio_mobilization_workshop/01-introduction/index.html) seems cleaner in comparison, but maybe with the right dropdowns the Carpentries format would be better.

**Comment by MathewBiddle** [comment]
Yeah, we can do that. The syntax would need to be updated in all of the pages. 

See our syntax at https://github.com/MathewBiddle/test_workshop/blob/2da056a90700478e60ba8080c2c49a7e26fb1978/episodes/01-introduction.md?plain=1#L30-L44

vs what the new syntax would look like
https://github.com/carpentries/sandpaper-docs/blob/6121b43fcd7df7a0123d9b41bcf8f515b92489cd/episodes/example.Rmd#L65-L93

I'll test this out and see what it looks like.

**Comment by MathewBiddle** [comment]
Take a look at https://mathewbiddle.github.io/test_workshop/01-introduction.html#darwin-core-archives

I only tested with that first challenge box.

Here's the code change https://github.com/MathewBiddle/test_workshop/commit/ecebb7210fa24f98508ba2589d6b99ba95bbc6f3

Seems easy enough of a fix to do to all of them.

**Comment by laurabrenskelle** [comment]
Yes, I like that better.

**Comment by MathewBiddle** [comment]
Okay, this page is fully updated. https://mathewbiddle.github.io/test_workshop/01-introduction.html

**Comment by MathewBiddle** [comment]
I've done some porting of the home page and the introduction. Let me know what you think:

* https://mathewbiddle.github.io/test_workshop/index.html
* https://mathewbiddle.github.io/test_workshop/01-introduction.html

The other episodes are in there but need to be updated for the new formatting. Before going down that road I'd like to know if it's worth pursuing.

**Comment by MathewBiddle** [comment]
TODO - fix dropdown content. for some reason it is being repeated and the numerical bullets aren't working as expected.

Otherwise, everything else has been ported into the new carpentries workbench format.

**Comment by MathewBiddle** [comment]
Forgot to add the link: https://mathewbiddle.github.io/test_workshop/index.html

**Comment by MathewBiddle** [comment]
this is done. we've migrated. See https://ioos.github.io/bio_mobilization_workshop/
